src/script/script.h

```diff
@@ -287,13 +291,19 @@ class CScriptNum {
     }
 
     inline CScriptNum operator/(const int64_t &rhs) const {
+        // Sanity check for invalid script num; should never occur
+        assert(m_value != std::numeric_limits<int64_t>::min());
+        assert(rhs != 0);
```

Comment:
What's the resulting error for these compared to existing behavior?

Reply:
I just added these asserts to make sure if there's any error in using these, the tests will blow up. Especially the -0x8000 0000 0000 0000 bug could be very subtle.